insight-engine/1.4/install/options.md

```diff
@@ -2,12 +2,9 @@
 title: Installation options
 ---
 
-There are several options for installing Search and Insight Engine:
+You can install Search and Insight Engine using the distribution zip or Docker Compose. If you use the distribution zip there are three different ways you can install the software, the first is with mutual TLS, the second is without mutual TLS (plain HTTP), and the third is without mutual TLS (HTTP with secret word in request header).
```

Comment:
Docker Compose is also supporting that three different ways

Reply:
adjusted sentence